<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Thembani Moithobogi</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>446</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>CFT Verification Tool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/cft.ico</normaloff>icon/cft.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>547</width>
     <height>420</height>
    </size>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>951</width>
      <height>421</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QGroupBox" name="outDirGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>525</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>525</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>525</width>
       <height>50</height>
      </size>
     </property>
     <property name="title">
      <string>Output Directory</string>
     </property>
     <widget class="QPushButton" name="outDirButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>93</width>
        <height>25</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="outDirPath">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>401</width>
        <height>25</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>380</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="zonesFileGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>525</width>
       <height>80</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>525</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>525</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Zones File (geojson)</string>
     </property>
     <widget class="QPushButton" name="zonesFileButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>93</width>
        <height>25</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="zonesFilePath">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>401</width>
        <height>25</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="zonesFileVariable_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Zone ID variable</string>
      </property>
     </widget>
     <widget class="QComboBox" name="zonesFileVariable">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="helpButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>380</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="logGroup">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>0</y>
       <width>390</width>
       <height>371</height>
      </rect>
     </property>
     <property name="title">
      <string>Run log</string>
     </property>
     <widget class="QPlainTextEdit" name="logWindow">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>390</width>
        <height>341</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="fcstGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>521</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Forecast period</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Year (of the first month):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Period:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="fcstPeriod">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLineEdit" name="fcstYear">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="clearLogButton">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>380</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Log</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="zoneGroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>521</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Zone properties</string>
     </property>
     <widget class="QLabel" name="signalAgree_label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Signal agreement</string>
      </property>
     </widget>
     <widget class="QLabel" name="skillLevel_label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Skill level</string>
      </property>
     </widget>
     <widget class="QComboBox" name="skillLevel">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="zoneCode_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Zone</string>
      </property>
     </widget>
     <widget class="QComboBox" name="zoneCode">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QComboBox" name="signalAgree">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QComboBox" name="fcstCategory">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>90</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="fcstCategory_label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Forecast category</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="confLevel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>120</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="confLevel_label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculated Confidence level</string>
      </property>
     </widget>
    </widget>
    <zorder>saveButton</zorder>
    <zorder>outDirGroup</zorder>
    <zorder>zonesFileGroup</zorder>
    <zorder>exitButton</zorder>
    <zorder>helpButton</zorder>
    <zorder>logGroup</zorder>
    <zorder>fcstGroup</zorder>
    <zorder>clearLogButton</zorder>
    <zorder>zoneGroup</zorder>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
